generator.js

```diff
@@ -1,92 +1,97 @@
 #! /usr/bin/env node
 
-const path = require('path')
-const { execSync } = require('child_process')
-const fs = require('fs-extra')
-const git = require('simple-git/promise')
-const commander = require('commander')
+const path = require('path');
+const { execSync } = require('child_process');
+const fs = require('fs-extra');
+const git = require('simple-git');
+const { program } = require('commander');
```

Comment:
aside from a few changes with version upgrades, and logging, nothing really changed with the generator, and this file can be fairly safely skimmed.